features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64LRUCache.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.recorder.base64
+
+import android.content.ComponentCallbacks2
+import android.content.res.Configuration
+import android.graphics.drawable.Drawable
+import android.util.LruCache
+import androidx.annotation.VisibleForTesting
+
+@Suppress("MagicNumber")
+internal object Base64LRUCache : Cache, ComponentCallbacks2 {
+    @Suppress("UnsafeThirdPartyFunctionCall") // maxMemory does not throw
```

Comment:
Do not `@Suppress` this warning, instead you need to add the `Runtime.getRuntime()` and `Runtime.maxMemory()` methods in the `detekt_custom.yml` in the `knownSafeCalls` section